### PR TITLE
Add OECS Flag

### PR DESCRIPTION
**Pull request by matmsa27**

### CODE REVIEW

File: ureport/settings_common.py [hunk 1]
```diff
@@ -1244,6 +1244,12 @@
         flag="flag_norge.png",
         count_link="http://norge.ureport.in/count/",
     ),
+    dict(
+        name="OECS",
+        host="//oecs.ureport.in",
+        flag="flag_oecs.png",
+        count_link="http://oecs.ureport.in/count/",
```
Comment (norkans7) [inline ureport/settings_common.py:1251]: To be accurate on the count of countries we changed to add the `countries_codes` as a list of countries alpha 3 ISO codes. See https://github.com/rapidpro/ureport/pull/645/files

We will deduplicate then to count the number of countries accurately

For this site I think it is not yet ready to be shown as we do not yet have a opinion to display

**Review by norkans7 — CHANGES_REQUESTED** [review]
Remove `static/img/flags/flag_oecs.png` too as that directory if no longer used

**Comment by matmsa27** [comment]
Hi @norkans7 , we adjusted the flag from OECS.

**Comment by norkans7** [comment]
screenshot?

File: ureport/settings_common.py [hunk 2]
```diff
@@ -1244,6 +1244,13 @@
         flag="flag_norge.png",
         count_link="http://norge.ureport.in/count/",
     ),
+    dict(
+        name="OECS",
+        host="//oecs.ureport.in",
+        flag="flag_oecs.png",
+        countries_codes=["ATG"],  # ["ATG", "VGB", "DMA", "GRD", "MSR", "KNA", "LCA", "VCT", "TCA"],
```
Comment (norkans7) [inline ureport/settings_common.py:1251]: is ATG the only country here?
Why do you keep the comment of the others?

Comment (matmsa27) [reply ureport/settings_common.py:1251]: OECS is a workspace that works with many countries. We setup the ATG and keep the others to keep on code the information that OECS is a workspace that work with other countries, like [Pacific](https://github.com/rapidpro/ureport/blob/main/ureport/settings_common.py#L1408)

**Comment by matmsa27** [comment]
> screenshot?

Clipboard - 21 de Junho de 2021 às 15_33

**Pull request merged**